data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/util/ContextMatcher.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.expression.util;
+
+import org.antlr.v4.runtime.tree.ParseTree;
+import org.hamcrest.Description;
+import org.hamcrest.DiagnosingMatcher;
+import org.hamcrest.Matcher;
+
+import javax.annotation.Nullable;
+
+import static org.hamcrest.CoreMatchers.isA;
+import static org.hamcrest.core.IsEqual.equalTo;
+
+/**
+ * @since 1.3
```

Comment:
As these are in the test source set, they are not shared with other projects. Thus, I think you don't need `@since`. But, it doesn't hurt to keep it.